be/src/storage/compaction_manager.cpp

```diff
@@ -57,6 +57,13 @@ void CompactionManager::update_candidates(std::vector<CompactionCandidate> candi
     _notify_schedulers();
 }
 
+void CompactionManager::update_max_threads(int max_threads) {
+    std::lock_guard lg(_scheduler_mutex);
+    for (auto&& s : _schedulers) {
```

Comment:
Why use && not &?

Reply:
&& is more universal. https://zhuanlan.zhihu.com/p/25148592